### DISCUSSION TITLE
Error: Resource not accessible by integration

### DISCUSSION THREAD

**Discussion by ghost** [upptime/upptime#926] [Q&A] [answered]
After uptading a file I get error
Run benc-uk/workflow-dispatch@v1
  with:
    workflow: Graphs CI
    token: ***
🏃 Workflow Dispatch Action v1.[2](https://github.com/Iliya5digital/upptime/actions/runs/7700771542/job/20985304873#step:6:2).1
🔎 Found workflow, id: 8412[3](https://github.com/Iliya5digital/upptime/actions/runs/7700771542/job/20985304873#step:6:3)313, name: Graphs CI, path: .github/workflows/graphs.yml
🚀 Calling GitHub API to dispatch workflow...
Error: Resource not accessible by integration

What to do?

**Comment by AnandChowdhary** [comment, marked as answer]
It looks like there are not sufficient permissions for workflow dispatching. Can you please make sure all the required scopes are added to your personal access token? https://upptime.js.org/docs/get-started#add-repository-secrets
